### ISSUE TITLE
skaffold build?

### ISSUE THREAD

**Issue by dgageot** [GoogleCloudPlatform/skaffold#220]
Would it make sense to have a `skaffold build` command?

**Comment by r2d4** [comment]
Yes I opened #35 awhile ago. I tried to autogenerate these from the build and deploy implementations, but that turned out to be a lot more trouble than it was worth. But yes, I'd like to see this exposed and return a BuildResult that can be templated with a go template, so folks could pipe it into their build system how they'd like.

**Comment by dgageot** [comment]
We might also want a `skaffold push`

**Comment by r2d4** [comment]
I think push could be a flag to skaffold build.

I imagine the CLI would look something like

`skaffold build -f skaffold.yaml --skipPush=true`

Where skaffold build would execute the build stanza of the existing skaffold.yaml and respect the skipPush parameter.

**Comment by viglesiasce** [comment]
I think i'd want `skaffold deploy` for the CI/CD side. We've add the --tag flag so in CI i would do:

skaffold deploy --tag v1.0.1
